docs/arcaflow/workflows/expressions.md

```diff
@@ -39,7 +41,51 @@ For example, to have the following text represented in a single string:
 
 You would need the expression `"test\ntest2/\\"`
 
-### Integer
+#### String-expressions in YAML
+
+If an expression has a string literal in it, then you must account for it in the YAML.
+
+For example, to include a double-quoted string in an expression, you must either add single quotes around expression, or use block flow scalars. A single apostrophe within an expression needs to be added twice in a row to count as one while in an inlined string.
+
+Here is an example of the following value represented in a few of the various ways:
+> just a string with sub-quotes "hello" and an apostrophe '
+
+Inlined:
+```
+some_value: !expr '"just a string with sub-quotes \"hello\" and an apostrophe \'' "'
+```
```

Comment:
Some additional explanation of this example under it would be good:
- Explain the purpose/use/requirement of `!expr`;
- Point out that the single quotes make YAML pass the contents intact except for replacing the repeated apostrophe with a single one (I know that you said this above, but saying it again under a concrete example is worthwhile);
- Point out that the escapes are removed by Arca's expression processing.  (Why is the single quote escaped?)